managed-apps-vision/README.md

```diff
@@ -0,0 +1,30 @@
+# Managed Apps Vision
+
+This RFC describes a general vision for improving the user experience of Managed Apps.
+
+With a previous iteration of the structure of Giant Swarm, one team was responsible for all of the managed apps. As the number of apps that Giant Swarm managed increased, it became more difficult to own these apps.
+
+Giant Swarm has updated its structure so that ownership of apps is spread across multiple teams - instead of one team owning all apps, each team owns a number of apps. Each team is now responsible for an area of solutions, such as monitoring, and are then responsible for apps that help in that area, such as Prometheus. This ownership also includes concepts such as trainings or sales enablement. We've been referring to this concept as 'holistic ownership'.
+
+With this change, more teams and more people are working on apps, overall. This means we can spend more time on each app, and provide a better user experience overall - we can start working more _deeply_.
```

Comment:
```suggestion
With this change, more teams and more people are working on apps, overall. This means we spend more time on each app, and provide a better user experience overall - we can work more _deeply_ on each solution.
```

Reply:
I don't know what the correct word is for the difference between these tenses, but I think cause this is a proposal for a change in the future, putting it as 'we _can_ spend more time' makes more sense, right? similarly with 'we can start working'

Reply:
Are our RFCs public?
The thing is that I have a preference for strong and single minded language, rather than something that creates loopholes or doubts.